nur/update.py

```diff
@@ -36,6 +36,7 @@ def eval_repo(repo: Repo, repo_path: Path) -> None:
             "-qa", "*",
             "--meta",
             "--xml",
+            "--allowed-uris", "https://static.rust-lang.org",
```

Comment:
I am only allowing rust-lang for now so we can get at least updates for the mozilla overlay.
I still could not get `--allowed-uris` to work with `allow-import-from-derivation`.